cpp/log/cert.cc

```diff
@@ -578,28 +578,28 @@ Cert::Status Cert::SPKISha256Digest(string* result) const {
 
   result->assign(sha256_digest);
   OPENSSL_free(der_buf);
-  return TRUE;
+  return util::Status::OK;
 }
 
-
-Cert::Status Cert::OctetStringExtensionData(int extension_nid,
-                                            string* result) const {
+StatusOr<bool> Cert::OctetStringExtensionData(int extension_nid,
```

Comment:
For this one, `false` isn't really used, `true` really means `OK`, and we might want to return `StatusOr<string>`?